src/engine/Settings.cs

```diff
@@ -111,7 +111,21 @@ public enum StrainBarVisibility
     ///   enabled.
     /// </summary>
     [JsonProperty]
-    public SettingValue<float> MicrobeDistortionStrength { get; private set; } = new(0);
+    public SettingValue<float> MicrobeDistortionStrength { get; private set; } = new(0.6f);
+
+    /// <summary>
+    ///   The amount of blur applied to microbe backgrounds when distortion is on.
```

Comment:
This comment seems to be duplicated from the below variable. So maybe this should be removed and blur strength of 0 should mean off?